src/main/java/com/cspark/nextstep/step3/domain/CarRace.java

```diff
@@ -0,0 +1,57 @@
+package com.cspark.nextstep.step3.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map.Entry;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class CarRace {
```

Comment:
Car의 컬렉션 객체로써 복수를 뜻하도록 네이밍해도 좋을 것 같아요 :)
```suggestion
public class Cars {
```